### PR TITLE
Bay Port - Mutual Language Understanding

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Ports over simple language understanding code. For an example of how this works:
	
Say a person speaks Serbian and the other speaks Russian. When speaking their languages, they have a 60% chance to understand **each** word from them. (I.E - they type "How are you today good sir?" - it may come up as "Vodka blyad cheeki good cyka?" or any combination as each word you type is a probability chance of 60 to understand)

Added a lot here, so I suggest reading the files changed per-langauge.
## Changelog
:cl:
add: Adds Bay's mutual understanding chance to languages. Ranging from 5% intelligibility between Latin and Common/English to 60% between Russian and Serbian.
/:cl:

### CODE REVIEW

File: code/modules/mob/language/serbian.dm [hunk 1]
```diff
@@ -302,7 +305,7 @@
 "Jerkov",
 "Jigovic",
 "Jorgic",
-"Karad�ic",
+"Karad�ic",
```
Comment (Trilbyspaceclone) [inline code/modules/mob/language/serbian.dm:308]: whats with breaking simbles?

File: code/modules/mob/language/serbian.dm [hunk 2]
```diff
@@ -268,7 +271,7 @@
 "Cubrilovic",
 "Cvijic",
 "Cvijovic",
-"�ajic",
+"�ajic",
```
Comment (Trilbyspaceclone) [inline code/modules/mob/language/serbian.dm:274]:
```suggestion
"Đajic",
```

File: code/modules/mob/language/serbian.dm [hunk 3]
```diff
@@ -288,7 +291,7 @@
 "Gracanin",
 "Grba",
 "Grlic",
-"Grmu�a",
+"Grmu�a",
```
Comment (Trilbyspaceclone) [inline code/modules/mob/language/serbian.dm:294]:
```suggestion
"Grmuša",
```

File: code/modules/mob/language/serbian.dm (hunk 1 above)
Comment (Trilbyspaceclone) [inline code/modules/mob/language/serbian.dm:308]:
```suggestion
"Karadžic",
```

Comment (Stutternov) [reply code/modules/mob/language/serbian.dm:309]: Guessing something to do with saving it on Visual Studio? Let me fix that.

Comment (ThePainkiller) [reply code/modules/mob/language/serbian.dm]: Will these characters show up with our encoding?

**Pull request merged**